File: tests/fixtures/adapters/elixir/negative/strings.ex
msg = "def not_a_function(x) do"
template = ~S(defmodule NotReal do)
pattern = ~r/defmacro\s+\w+/
code = "defmodule Ghost do\n  def run do\n  end\nend"
heredoc_first = "def nope do"

File: tests/fixtures/adapters/javascript/negative/strings.js
// Export-looking text inside strings must NOT be flagged,
// because none of these lines begin with `export ` / `module.exports`.
const msg = "export function fakeExport() {}";
const tpl = `export const notReal = 42`;
const code = "module.exports = function nope() {}";

function logger() {
  return "export default function pretend() {}";
}
